k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/service/IgorArtifactService.kt

```diff
@@ -0,0 +1,49 @@
+package com.amazon.spinnaker.keel.k8s.service
+
+import com.amazon.spinnaker.keel.k8s.model.GitVersion
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.config.DefaultServiceEndpoint
+import com.netflix.spinnaker.config.okhttp3.OkHttpClientProvider
+import com.netflix.spinnaker.keel.retrofit.InstrumentedJacksonConverter
+import okhttp3.HttpUrl
+import retrofit2.Retrofit
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface IgorArtifactRestService {
+    @GET("/git/version/{type}/{project}/{slug}")
+    suspend fun getGitVersions(
+        @Path("type") type: String,
```

Comment:
Curious what is the `type` here? looks like we pass `git` as the type, but then `git` appears in the first part of the URL also. can we drop the redundancy here?

Reply:
`type` is the managed git provider. It could be github, gitlab, bitbucket, etc. Currently only GitHub is supported however.